hackathon_site/event/api_urls.py

```diff
@@ -14,4 +14,7 @@
     ),
     path("teams/leave_team/", api_views.LeaveTeamView.as_view(), name="leave-team"),
     path("teams/", views.TeamListView.as_view(), name="team-list"),
+    path(
+        "teams/<str:team_code>", views.TeamCodeView.as_view(), name="team-code-detail"
```

Comment:
We'll have a discussion about this today during the meeting, but I think this should be `teams/<int:pk>/` to do the lookup based off of primary key (which is `id`).

I think the use case for this endpoint is to eg fetch a team given an order response. The order response contains just the team's ID, not their team code, so that's what we need the lookup to be based off of if that is the intended use case.

Reply:
```suggestion
        "teams/<str:team_code>", views.TeamCodeView.as_view(), name="team-detail"
```